ha/setup/ha_setup.py

```diff
@@ -428,6 +428,12 @@ def _add_node(self, node_name: str, cluster_user: str, cluster_secret: str) -> N
             add_node_cli = const.CORTX_CLUSTER_NODE_ADD.replace("<node>", node_name)\
                 .replace("<user>", cluster_user).replace("<secret>", cluster_secret)
             self._execute.run_cmd(add_node_cli, check_error=False, secret=cluster_secret)
+            node_status =  self._cluster_manager.cluster_controller.wait_for_node_online(node_name)
+            # Nodes are added to the cluster during cluster creation
+            # Node is expected to be Online when it is added
+            if node_status != True:
+                raise Exception
```

Comment:
Should add some meaningful exception.

Reply:
Okay, Will add HaConfigException  here .

Reply:
Done